webapp/templates/partials/scoreboard_table.html.twig

```diff
@@ -135,7 +135,10 @@
         {% endif %}
         <tr class="{{ classes | join(' ') }}" data-team-id="{{ score.team.teamid }}">
             {% if enable_ranking %}
-                <td class="scorepl {{medalColor}}">
+                <td class="scorepl">
+                    {% if medalColor != '' %}
+                        <i class="fa fa-medal {{medalColor}}" style="font-size:150%;"></i>
```

Comment:
I'm not 100% sure but is it ok that we mix `em` and `%` throughout the code?

Reply:
Good point, using `rem` instead now.